src/app/_components/KeydeukBest/KeydeukBest.tsx

```diff
@@ -0,0 +1,48 @@
+'use client';
+
+import { getKeydeukBest } from '@/api/productAPI';
+import ProductItem from '@/components/Products/ProductItem';
+import { QUERY_KEYS } from '@/constants/queryKey';
+import { useQuery } from '@tanstack/react-query';
+import classNames from 'classnames/bind';
+import 'swiper/css';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import { Autoplay } from 'swiper/modules';
+import styles from './KeydeukBest.module.scss';
+
+const cn = classNames.bind(styles);
+
+export default function KeydeukBest() {
+  const { data } = useQuery({
+    queryKey: QUERY_KEYS.PRODUCT.LISTS(),
+    queryFn: getKeydeukBest,
+  });
+  return (
```

Comment:
```suggestion
  const { data: data for what? } = useQuery({
    queryKey: QUERY_KEYS.PRODUCT.LISTS(),
    queryFn: getKeydeukBest,
  });
  
  return (
```
data를 따로 지정해서 사용하시는게 더 좋은 사용방법으로 알고있어요